sormas-cargoserver/build_deploy.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Execute with ./build_deploy.sh <arg> , e.g. ./build_deploy.sh fast
+
+mvn cargo:stop
+
+# recompile
+cd ../sormas-base || exit
+
+if [ "$1" = "wipe" ]; then
+  cd ../sormas-cargoserver || exit
+  docker-compose down
+  docker volume rm sormas-cargoserver_psqldata_cargoserver
+  docker-compose up -d
+  cd ../sormas-base || exit
+  mvn clean install -DskipTests
```

Comment:
Wouldn't the inverse order (first mvn clean install, then docker-compose up) make more sense?

Reply:
hm I don't think it makes much of a difference :D